packages/angular_devkit/build_optimizer/src/transforms/class-fold.ts

```diff
@@ -6,7 +6,6 @@
  * found in the LICENSE file at https://angular.io/license
```

Comment:
@clydin and @filipesilva should we remove the class-fold ES5 logic completely. Ie depreciate it and not use it?

I tried this and it seems that static properties are always folding an in an IIFE by typescript at least for version 2.0+

Reply:
I think we should deprecate, but not remove yet. It's still part of the public API.